lib/public/views/Runs/formatRunDuration.js

```diff
@@ -27,16 +27,18 @@ export const formatRunDuration = (run, raw = false) => {
     if (runDuration === null || runDuration === undefined) {
         return '-';
     } else if (timeTrgEnd || timeO2End) {
-        return `${Math.floor(runDuration / 60000)} min${raw ? '' : getAsteriks(run)}`;
+        return `${formatDuration(runDuration)}${raw ? '' : _getAsteriks(run)}`;
     } else {
-        const time = timeTrgStart ? timeTrgStart : timeO2Start;
+        const timeStart = timeTrgStart == null || timeTrgStart === undefined ?
```

Comment:
triple equal for null check or second check is always false

Reply:
Fixed